testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala

```diff
@@ -99,4 +100,14 @@ class AdminConsoleSimulation extends Simulation {
   setUp(adminScenario
     .inject(rampUsers(TestConfig.runUsers) over TestConfig.rampUpPeriod)
     .protocols(httpProtocol))
+
+  after {
+    if (TestConfig.filterResults) {
+      new LogProcessor(getClass).filterLog(
+        TestConfig.measurementStartTime, 
+        TestConfig.measurementEndTime,
+        false, true, true)
+    }
+  }
+
```

Comment:
Maybe we could move this code to CommonScenarioBuilder and only have here something like:
```
after {
  filerResults(getClass)
}
```

Reply:
Moved.